feat(vuln): enable `--vex` for all targets (#5992)
aquasecurity/trivy: docs/docs/supply-chain/vex.md

CVE-2019-8457 is no longer shown as it is filtered out according to the given OpenVEX document.

## CSAF
|     Target      | Supported |
|:---------------:|:---------:|
| Container Image |     ✅     |
|   Filesystem    |     ✅     |
| Code Repository |     ✅     |
|    VM Image     |     ✅     |
|   Kubernetes    |     ✅     |
|      SBOM       |     ✅     |

Trivy also supports [CSAF][csaf] format for VEX.
Since CSAF aims to be SBOM format agnostic, both CycloneDX and SPDX formats are available for use as input SBOMs in Trivy.

The following steps are required:

> 1. Create a CSAF document based on the SBOM generated in step 1
> 2. Provide the CSAF document when scanning the SBOM

Reviewer: ```suggestion
1. Create a CSAF document
2. Provide the CSAF when scanning your target

```

Author: Thanks. Fixed.
